PicoStreamingAssistantFTUDP/Pico4SAFTExtTrackingModule.cs

```diff
@@ -27,18 +28,66 @@ public sealed class Pico4SAFTExtTrackingModule : ExtTrackingModule, IDisposable
 
     private const bool FILE_LOG = false;
     public static readonly string LOGGER_PATH = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "VRCFaceTracking\\PICOLogs.csv");
-    private Logger<PxrFTInfo> logger;
+    private Logger<PxrFTInfo>? logger;
 
     public override (bool SupportsEye, bool SupportsExpression) Supported { get; } = (true, true);
 
+
     private bool StreamerValidity()
     {
-        if (Process.GetProcessesByName("Streaming Assistant").Length is 0)
+        if (!(Process.GetProcessesByName("Streaming Assistant").Length is 0))
+        {
+            PORT_NUMBER = 29765;
+            Logger.LogInformation("Processes \"Streaming Assistant\" founded");
+            return true;
+        }
+        else if (!(Process.GetProcessesByName("PICO Connect").Length is 0))
         {
-            Logger.LogError("\"Streaming Assistant\" process was not found. Please run the Streaming Assistant before VRCFaceTracking.");
+            Logger.LogInformation("Processes \"PICO Connect\" founded");
+            string configLocation = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "PICO Connect\\settings.json");
+            string configTemp = string.Empty;
+            Config? picoConfig;
+            try
+            {
+                configTemp = File.ReadAllText(configLocation);
+                picoConfig = JsonSerializer.Deserialize<Config>(configTemp);
+            }
+            catch (Exception ex)
+            {
+                Logger.LogError("Pico Connect Config Deserialize Failed");
+                return false;
+            }
+
+            if (picoConfig?.lab.faceTrackingTransferProtocol == 1)
+            {
+                PORT_NUMBER = 29763;
```

Comment:
Did you check if just changing the port works? I was in charge of the migration from v1 to v2 and they did change more things apart from the port (like the header of the packets). You could keep compatibility of both ( you can use my old PR as reference, https://github.com/regzo2/PicoStreamingAssistantFTUDP/pull/6 ), but as this adds extra maintenance I'd rather drop support of older versions.